components/style/values/computed/border.rs

```diff
@@ -81,3 +82,30 @@ impl ToAnimatedZero for BorderCornerRadius {
         Err(())
     }
 }
+
+/// The computed value of the `border-image-repeat` property:
+///
+/// https://drafts.csswg.org/css-backgrounds/#propdef-border-image-repeat
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToCss)]
+pub struct BorderImageRepeat(pub RepeatKeyword, pub RepeatKeyword);
```

Comment:
We already have a `RepeatKeyword` in the `background` module, maybe rename that one as `BackgroundRepeatKeyword`, and this as `BorderImageRepeatKeyword` as a followup?

Reply:
`(Background)RepeatKeyword` appeared in gecko.mako.rs in two places before my changes. The first one is clearly rename-able, but the second one is in `fn to_servo(repeat: StyleImageLayerRepeat) -> RepeatKeyword`. Is `RepeatKeyword` being used here in the background sense, or is it being reused for some other purpose? Does this affect the renaming decision? Either way, I've renamed the `RepeatKeyword` I just added to `BorderImageRepeatKeyword`.

Reply:
`ImageLayerRepeat` is definitely related to backgrounds in Gecko terminology, so I think it's fine to rename both.